src/engine/Filter.cpp

```diff
@@ -521,6 +527,19 @@ void Filter::computeResultFixedValue(
       } else if (ad_utility::isNumeric(_rhs)) {
         rhs_string = ad_utility::convertNumericToIndexWord(rhs_string);
       } else {
+        // TODO: This is not standard conform, but currently required due to
+        // our vocabulary storing iris with the greater than and
+        // literals with their quotation marks.
+        if (rhs_string.find('<') != std::string::npos &&
+            rhs_string.back() == '"' && rhs_string.size() > 1) {
```

Comment:
I'm a little worried that this check isn't strict enough, maybe we should only strip the quotes for `"<…"` with no space between `"` and `<`

Reply:
That seems like a good idea. I've made the check more strict.